Demos/Blazorise.Demo/Pages/Tests/ChartsDataLabelsPage.razor.cs

```diff
@@ -182,10 +182,12 @@ public partial class ChartsDataLabelsPage
                 BackgroundColor = BackgroundColors[2],
                 BorderColor = BorderColors[2],
                 Align = "center",
-                Anchor = "center"
+                Anchor = "center",
+                ScriptableFormatter = ScriptableFormatter
             }
         },
     };
+    static Expression<Func<object, ScriptableOptionsContext, string>> ScriptableFormatter = ( value, context ) => "$ " + value;
```

Comment:
`Lamda2JS` isn't the most intuitive library to use as there is a lot that isn't supported by default so anyone wanting to use this will have to be  careful with the code they write. 

I went through a lot of trial and error by adding a having it generating my javascript file [here ](https://github.com/Megabit/Blazorise/pull/5146/files#diff-ee58a9f6c938323926d359b9a0f6d164d0f4a54bd8af138de23e42c51ef56254R56)and then testing it manually in a browser to find my errors